src/utils/Route.ts

```diff
@@ -6,10 +6,15 @@ const ROUTE_PATH_LoginPage = "/login";
 const ROUTE_PATH_SignupPage = "/signup";
 const ROUTE_PATH_ProfilePage = "/profile";
 const ROUTE_PATH_Mockman = "/mockman";
+
+
 const VAR_ENCODE_TOKEN = "NotesifyToken";
 const VAR_USER_ID = "NotesifyUserId";
 const VAR_USER_DETAILS = "NotesifyUserDetails";  
 
+
+const VAR_PinnedNotes = "pinned";
+const VAR_NotPinnedNotes = "notpinned";
```

Comment:
Good use of constant variables. You add more variables here and avoid the hardcoded strings. Also,  please do make  proper readable variables as mentioned in previous comments.